staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/envelope.go

```diff
@@ -142,13 +180,40 @@ func (t *envelopeTransformer) TransformToStorage(data []byte, context value.Cont
 	// Append the length of the encrypted DEK as the first 2 bytes.
 	b := cryptobyte.NewBuilder(nil)
 	b.AddUint16LengthPrefixed(func(b *cryptobyte.Builder) {
-		b.AddBytes([]byte(encKey))
+		b.AddBytes(dekCiphertext)
 	})
 	b.AddBytes(result)
 
 	return b.Bytes()
 }
 
+func (t *envelopeTransformer) rotate() ([]byte, value.Transformer, error) {
+	var (
```

Comment:
don't use a var block and declare the err close to where it's used

Reply:
Done.